BuiltDifferentMobileApp/BuiltDifferentMobileApp/BuiltDifferentMobileApp.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
+    <UserSecretsId>16650aef-dcc0-4eab-ae33-9d1d20305c2f</UserSecretsId>
```

Comment:
just curious on what this might be.

Reply:
I am not sure as well, it might have happened when I restored the nuget packages